packages/client/src/pages/pinia.vue

```diff
@@ -67,10 +73,10 @@ onDevToolsClientConnected(() => {
         </div>
       </Pane>
       <Pane flex flex-col>
-        <div h-0 grow overflow-auto p-2 class="no-scrollbar">
+        <div :key="selected" h-0 grow overflow-auto p-2 class="no-scrollbar">
           <InspectorState
             v-for="(item, key) in state" :id="key"
-            :key="key + Date.now()"
```

Comment:
When the state is updated, the whole InspectorState component tree was fully re-rendered.